Task3

patrikprovaznik opened the pull request.

patrikprovaznik committed:
adding config file

patrikprovaznik committed:
adding utils.py with get_logger function

patrikprovaznik committed:
adding utils.py with get_logger function skuska

patrikprovaznik committed:
adding interface_manager.py file

patrikprovaznik committed:
1. adding config file for interface_api.py(server side)

patrikprovaznik committed:
adjustments that have been made:
1. we created config file flask_app_config.ini which is mostly used in flask app but also in client app as default value if args do not exist
2. secret key is not in config file anymore
3. we created requirements.txt file and readme.md
4. removing unnecessary files from task3 (these files were replaced with correct files by pulling from git)

mirokovac commented on task1/flask_app_config.ini:
```diff
@@ -0,0 +1,16 @@
+secret_key =
+
+[general]
+logs_dir = logs/
+log_path_1 = logs/logger_1.log
```
@patrikprovaznik try and use interpolation rather the repeating path -> https://docs.python.org/3/library/configparser.html#interpolation-of-values

mirokovac commented on task1/flask_app_config.ini:
```diff
@@ -0,0 +1,16 @@
+output_folder = output_folder/
+output_data = output_data.json
+log_data = log_data.json
+log_path_2 = logs/logger_2.log
```
same as above

mirokovac commented on task2/main.py:
```diff
@@ -1,16 +1,22 @@
 def main(**kwargs) -> None:
+    # config
+    conf = ConfigParser()
+    conf.read("/home/patrik/Documents/patprotasks/task1/flask_app_config.ini")
```
@patrikprovaznik dont use absolute path in the program.. this changes based on the user.. either read it from user input or use absolute path if the project is suppose to contain this in some resources folder

mirokovac reviewed (changes requested):
@patrikprovaznik just some quick small issues to fix othewise looks good

patrikprovaznik committed:
adjustments that have been made:
1. adding interpolation of values into config
2. removing absolute path from main.py
3. adding config into resource folder, also relocating resource folder to root directory
